docker/Dockerfile

```
@@ -1,20 +1,16 @@
-FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu16.04
+FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu16.04 AS base
 
-MAINTAINER masuda@leapmind.io
+MAINTAINER admin@blueoil.org
```

MAINTAINER is deprecated. Please use LABEL.
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated
```suggestion
LABEL maintainer="admin@blueoil.org"
```

---

fixed